app/models/user.rb

```diff
@@ -11,6 +11,8 @@ class User < ApplicationRecord
     if: :address_changed?
 
   after_validation :geocode, :reverse_geocode
+  after_create :create_usersetting
```

Comment:
**Reporter**: CHECKSTYLE
**Rule**: com.puppycrawl.tools.checkstyle.Style/AlignHash
**Severity**: INFO
**File**: app/models/user.rb L14

Align the elements of a hash literal if they span more than one line.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>*